api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/Flags.java

```diff
@@ -907,4 +907,185 @@ public String uuid() {
             return "8311e3eb-d87d-40f2-a35f-dd60f1782ddd";
         }
     };
+    public static FlagDescriptor PATIENT_HTN_STATUS = new FlagDescriptor() {
```

Comment:
We should not have flags targeting the observation table. Switch statement to target mamba tables